cms_delete_content/static/src/js/delete_content.js

```diff
@@ -0,0 +1,58 @@
+odoo.define('cms_delete_content.delete_confirm', function (require) {
+    'use strict';
+
+    var ajax = require('web.ajax');
+    var msg_tool = require('cms_status_message.tool');
+    var core = require('web.core');
+    var _t = core._t;
+
+    $(document).ready(function () {
+      $(".cms_delete_confirm").click(function(evt){
```

Comment:
well, this is a 1st prototype and I could make it more extendable even without animation. Something like this https://github.com/simahawk/website-cms/blob/ae9819c1b64d86eaef4aa804749b811bd30a64be/cms_status_message/static/src/js/tool.js#L27
Which are the real benefits for using an animation (as long as this is not something you are going to edit w/ website builder)?